### ISSUE TITLE
[Bug Report] Sorting Scenes by Title yields Status Code 422 Message

### ISSUE THREAD

**Issue by Cygon** [stashapp/stash#1572]
**Describe the bug**
When I sort my scenes by "Title", I am seeing the following message:

```Response not successful: Received status code 422```

This error started without any update to the Stash application. I first noticed it with Stash 0.7.0 and initially, could go past it by simply hitting the reload button in my browser. That stopped working at some point. Upgrading to Stash 0.8.0 didn't change the issue.

The log shows ```json: error calling MarshalJSON for type json.RawMessage: invalid character '+' looking for beginning of value``` at the same time as the status code message.

**To Reproduce**
1. Select the "Scenes" tab
2. Change the default "Date" search criterion to "Title"
3. (probably needs a secret ingredient, i.e. a movie with a problematic filename)
4. Observe status code 422 being reported

**Expected behavior**
The scenes should be listed as usual, sorted by their "Title" field

**Screenshots**
![stash-status-code-422](https://user-images.githubusercontent.com/3613175/125335776-3d939080-e34d-11eb-97bf-227660b1c596.png)

![stash-error-422-logs](https://user-images.githubusercontent.com/3613175/125335872-58660500-e34d-11eb-8833-9b021e667d24.png)

**Stash Version: (from Settings -> About):**
v0.8.0 0x417ad4 2021-07-01 23:59:02

**Desktop (please complete the following information):**
 - Gentoo Linux (stable, amd64) running as LXC container under QNAP Linux kernel 4.14.200
 - Firefox 78.10.1esr (amd64)

**Smartphone (please complete the following information):**
*nope*

**Additional context**
Since the logs show a JSON issue with a '+' character, I tried searching for this specific character. Entering a plus character in the search box does not reproduce the error and lists 12 scenes.

**Comment by jocphi** [comment]
I had the same error coming up on a scene (I used number of pages and pagenumber in url to find the entry that wouldnt show up).

I then used DB-browser (https://github.com/sqlitebrowser/sqlitebrowser) to take a look at the entry causing this error. The only difference to all the other scenes in the database was - the framerate was set to "Inf.". After resetting the framerate to "NULL" the scene didnt break the pageview.

The video codec on that particular file was vp6f.

Maybe something to look into?

**Comment by Cygon** [comment]
I tried that and found one scene, imported from a .flv file with framerate "inf". I just wrote "30" in there because I didn't see any NULL rows.

ffprobe shows this for the file in question:
```
    Stream #0:0: Audio: adpcm_swf, 22050 Hz, stereo, s16, 176 kb/s
    Stream #0:1: Video: flv1, yuv420p, 448x336, 25 tbr, 1k tbn
```

However, that didn't seem to change anything (I stopped stash, replaced the database, started it again and Shift+Reloaded the browser, just in case).

I also noticed many rows had an empty strings in the details and url columns and replaced those with NULL.

Surprisingly, after that change, I can navigate my scene directory again.
